tools/check_metfile.py

```diff
@@ -0,0 +1,137 @@
+               'Fout in metinginformatie']
+    resultaten_dict = dict((each, []) for each in headers)
+
+    # Maak een list om de codes van alle profielen op te slaan
+    codes_list = []
+
+    # Opening metfile en data inlezen
+    with open (input_file, 'r') as metfile:
+        # Maakt de info als 1 lange text, handig om in te zoeken
+        met = metfile.read().replace('\n', '').replace('\r', '')
+
+        # Split de data op profielniveau
+        profielen = met.split('<PROFIEL>')
+
+        # Ga elk profiel af
+        for each in profielen[1:]:
+            profiel_elementen_totaal = each.split(',')
+            profielnaam = profiel_elementen_totaal[0]
+
+            # Checkparameter of er een fout is, dus het profiel naar de output moet worden geschreven
+            fout_aanwezig = False
+
+            # Maak list om de fouten van dit profiel op te slaan
+            resultaten_list = ['-'] * len(headers)
+            resultaten_list[0] = profielnaam
+
+            # Check of elk profiel ook een afsluitcommand heeft
+            if profiel_elementen_totaal[-1] != '</METING></PROFIEL>':
+                resultaten_list[1] = 'ja'
+                fout_aanwezig = True
+                #print 'Afsluiting missend: </PROFIEL>'
+
+            # Check of er fouten zitten in de profielinfo, normaal gezien zou het 10e element de eerste meting bevatten,
+            # zo niet dan zit er hoogstwaarschijnlijk een fout in de info
+            if len(profiel_elementen_totaal) < 10:
+                resultaten_list[2] = 'ja'
+                fout_aanwezig = True
+                #print 'Fout in profielinfo'
+            elif profiel_elementen_totaal[10].find('<METING>') < 0:
+                resultaten_list[2] = 'ja'
+                fout_aanwezig = True
+                #print 'Fout in profielinfo'
+
+            # Split het profiel op metingniveau
+            metingen = each[1:].split('<METING>')
+            # Check of er fouten zitten in de metingen, ga elk meetpunt af
+            for meetpunt in metingen[1:]:
+                meetpunt_elementen = meetpunt.split(',')
+
+                # Check of elk meting ook een afsluitcommand heeft
+                if meetpunt_elementen[-1]!= '</METING>' and meetpunt_elementen[-1]!= '</METING></PROFIEL>':
+                    resultaten_list[3] = 'ja'
+                    fout_aanwezig = True
+                    #print  'Afsluiting missend: </METING>'
+
+                # Check op het aantal elementen in de meting, normaliter zijn dat er 6
+                aantal_elementen =  len(meetpunt_elementen)-1 # de </meting> is ook een element, vandaar -1
+                if aantal_elementen != 6:
+                    resultaten_list[4] = 'ja'
+                    fout_aanwezig = True
+                    #print 'Fout in meting'
+
+                # Check of de elementen wel de juiste waarde/format hebben
+                for ind, e in enumerate(meetpunt_elementen[:-1]):
```

Comment:
ind en e? 
voor tellers wordt vaak i, ii of nr gebruikt. e zou ik 'element' of 'waarde' van maken of zoiets.
e wordt vaak gebruikt bij exception, beter om dat dus niet te gebruiken
````
try:
   ...
except IOError as e:
  print(e.message)
```